.gitignore

```diff
@@ -5,3 +5,4 @@
 /node_modules/
 allure-report
 .xml
+package.json
```

Comment:
why?

Reply:
It was decided at yesterday's lecture, and made under the guidance of Julia,  I was sharing my screen at that time.

Reply:
Still thinking about it. Haven't decided how to prevent conflicts when each team member will add his own test script.